src/Resources/contao/templates/backend/be_login.html5

```diff
@@ -81,7 +81,7 @@
         <p>Contao Open Source CMS :: Copyright ©2005-<?= date('Y') ?> by Leo Feyer :: Extensions are copyright of their respective owners :: Visit <a href="https://contao.org" target="_blank">contao.org</a> for more information :: Obstructing the appearance of this notice is prohibited by law!</p>
         <p>Contao is distributed in the hope that it will be useful but WITHOUT ANY WARRANTY. Without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details. Contao is free software. You can redistribute it and/or modify it under the terms of the GNU/LGPL as published by the Free Software Foundation.</p>
       </div>
-      <p id="go_to_frontend"><a href="<?= $this->route('contao_root') ?>" class="footer_preview" title="<?= $this->feLink ?>"><?= $this->feLink ?></a></p>
```

Comment:
I already consider it a ramification that you cannot use `$this->route('contao_root')` anymore. And `/` is a valid URL, even if `prepend_locale` is enabled. There is a "do not redirect the empty URL" option in the back end settings!